doc/src/fix_wall_gran.rst

```diff
@@ -181,11 +185,35 @@ info on how to re-specify a fix in an input script that reads a
 restart file, so that the operation of the fix continues in an
 uninterrupted fashion.
 
-None of the :doc:`fix_modify <fix_modify>` options are relevant to this
-fix.  No global or per-atom quantities are stored by this fix for
-access by various :doc:`output commands <Howto_output>`.  No parameter
-of this fix can be used with the *start/stop* keywords of the
-:doc:`run <run>` command.  This fix is not invoked during :doc:`energy minimization <minimize>`.
+If the :code:`store_contacts` option is used, this fix will store the contact
```

Comment:
This should say the output is a per-atom array with 8 columns,
whose values are listed in the table.